atd-vze/src/views/Crashes/crashDataMap.js

```diff
@@ -45,7 +45,7 @@ export const createCrashDataMap = isTempRecord => {
         },
         fhe_collsn_id: {
           label: "Manner of Collision ID",
-          editable: false,
+          editable: isTempRecord,
```

Comment:
Cool to be reusing this code from https://github.com/cityofaustin/atd-vz-data/pull/1238!